<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Система коллективного интеллекта</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="title">
       <string>Меню</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_6">
         <property name="title">
          <string>Файлы</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="leftMargin">
           <number>6</number>
          </property>
          <property name="topMargin">
           <number>6</number>
          </property>
          <property name="rightMargin">
           <number>6</number>
          </property>
          <property name="bottomMargin">
           <number>6</number>
          </property>
          <item>
           <widget class="QPushButton" name="btnLoadFromFile">
            <property name="text">
             <string>Загрузить из файла</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnGenerate">
            <property name="text">
             <string>Сгенерировать</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_7">
         <property name="title">
          <string>Одиночные акторы</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="leftMargin">
           <number>6</number>
          </property>
          <property name="topMargin">
           <number>6</number>
          </property>
          <property name="rightMargin">
           <number>6</number>
          </property>
          <property name="bottomMargin">
           <number>6</number>
          </property>
          <item>
           <widget class="QCheckBox" name="cbSingleIp">
            <property name="text">
             <string>Интеллектуальный потенциал</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Сложность задачи</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="dsbBeta">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="readOnly">
               <bool>false</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
              <property name="decimals">
               <number>6</number>
              </property>
              <property name="minimum">
               <double>-12.000000000000000</double>
              </property>
              <property name="maximum">
               <double>12.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.100000000000000</double>
              </property>
              <property name="value">
               <double>-1.750000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Количество итераций</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="sbSingleCountIt">
              <property name="minimum">
               <number>100</number>
              </property>
              <property name="maximum">
               <number>100000</number>
              </property>
              <property name="singleStep">
               <number>100</number>
              </property>
              <property name="value">
               <number>200</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="btnComputeSingleIP">
            <property name="text">
             <string>Вычислить</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_8">
         <property name="title">
          <string>Групповой актор</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="leftMargin">
           <number>6</number>
          </property>
          <property name="topMargin">
           <number>6</number>
          </property>
          <property name="rightMargin">
           <number>6</number>
          </property>
          <property name="bottomMargin">
           <number>6</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Сложность задачи</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="dsbGroupBeta">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="readOnly">
               <bool>false</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
              <property name="decimals">
               <number>6</number>
              </property>
              <property name="minimum">
               <double>-12.000000000000000</double>
              </property>
              <property name="maximum">
               <double>12.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.100000000000000</double>
              </property>
              <property name="value">
               <double>-1.750000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Количество итераций</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="sbGroupCountIt">
              <property name="minimum">
               <number>100</number>
              </property>
              <property name="maximum">
               <number>100000</number>
              </property>
              <property name="singleStep">
               <number>100</number>
              </property>
              <property name="value">
               <number>10000</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QRadioButton" name="rbGroupActorChecked">
              <property name="text">
               <string>Рассчитать для группы из выделенных</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="btnUncheck">
              <property name="text">
               <string>Ø</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QRadioButton" name="rbGroupActorAuto">
              <property name="text">
               <string>Подобрать группу автоматически,  m =</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="sbGroupSize">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="readOnly">
               <bool>false</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::UpDownArrows</enum>
              </property>
              <property name="minimum">
               <number>3</number>
              </property>
              <property name="maximum">
               <number>15</number>
              </property>
              <property name="singleStep">
               <number>2</number>
              </property>
              <property name="value">
               <number>3</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Qr = </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="dsbGroupQr">
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
              <property name="decimals">
               <number>6</number>
              </property>
              <property name="maximum">
               <double>1000000.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Qn = </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="dsbGroupQn">
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
              <property name="decimals">
               <number>6</number>
              </property>
              <property name="maximum">
               <double>1000000.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>IP = </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="dsbGroupIP">
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
              <property name="decimals">
               <number>6</number>
              </property>
              <property name="maximum">
               <double>1000000.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="btnComputeGroupIP">
            <property name="text">
             <string>Вычислить</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabMain">
       <attribute name="title">
        <string>База экспертов</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="leftMargin">
         <number>6</number>
        </property>
        <property name="topMargin">
         <number>6</number>
        </property>
        <property name="rightMargin">
         <number>6</number>
        </property>
        <property name="bottomMargin">
         <number>6</number>
        </property>
        <item>
         <widget class="TableActorsBase" name="tableActorsBase" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabGraphs">
       <attribute name="title">
        <string>Графики</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="leftMargin">
         <number>6</number>
        </property>
        <property name="topMargin">
         <number>6</number>
        </property>
        <property name="rightMargin">
         <number>6</number>
        </property>
        <property name="bottomMargin">
         <number>6</number>
        </property>
        <item>
         <widget class="GraphPlotter" name="plotter" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>480</width>
            <height>330</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="TableActorsBase" name="tableGroup" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>180</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>180</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TableActorsBase</class>
   <extends>QWidget</extends>
   <header>gui/TableActorsBase.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GraphPlotter</class>
   <extends>QWidget</extends>
   <header>gui/GraphPlotter.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
